src/rate.rs

```diff
@@ -1471,6 +1474,12 @@ impl RCState {
     );
     cur_pos =
       self.buffer_val(self.prev_metrics.log_scale_q24 as i64, 4, cur_pos);
+    eprintln!(
+      "show_frame {} fti {} log_scale_q24 {}",
+      self.prev_metrics.show_frame,
```

Comment:
Maybe have:
```rust
let prev_metrics = self.prev_metrics;
```
in the beginning of the function to make it a bit less verbose?

Reply:
it is a leftover, thanks for spotting. It will be removed.